src/harness/sas_testcase.py

```diff
@@ -100,3 +103,13 @@ def assertRegisteredAndGranted(self, registration_request, grant_request,
 
     # Return cbsd_ids and grant_ids
     return list_of_cbsd_ids_and_grant_ids
+
+  def TriggerDailyActivitiesImmediatelyAndWaitUntilComplete(self):
+    self._sas_admin.TriggerDailyActivitiesImmediately()
+    signal.signal(signal.SIGALRM, timeout_error)
```

Comment:
Huh! Looks like you technically *can* do it but it's so unreadable that it's probably not worth it: https://stackoverflow.com/questions/8294618/define-a-lambda-expression-that-raises-an-exception

That said, consider moving the Timeout stuff to this file if you're not going to use it anywhere else.